tests/playbooks/tests_ipv6_dns_search.yml

```diff
@@ -0,0 +1,93 @@
+---
+- hosts: all
+  tasks:
+    - import_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: br-example
+            type: bridge
+            ip:
+              dhcp4: false
+              auto6: false
+              dns: 8.8.8.8
+              dns_search: google.com
+              address:
+                - 10.2.2.77/26
+            state: up
+    - block:
+        - name: Verify nmcli connection DNS search entry for IPv4
+          shell: |
+            set -euxo pipefail
+            nmcli -f ipv4.dns-search connection show br-example
+          register: ipv4_dns_search
+          ignore_errors: yes
+          changed_when: false
+
+        - name: Verify nmcli connection DNS search entry for IPv6
+          shell: |
+            set -euxo pipefail
+            nmcli -f ipv6.dns-search connection show br-example
+          register: ipv6_dns_search
+          ignore_errors: yes
+          changed_when: false
+
+        - name: Assert that ipv4 DNS search entry is configured correctly
+          assert:
+            that:
+              - "'google.com' in ipv4_dns_search.stdout"
+            msg: "ipv4 DNS search entry is configured incorrectly"
```

Comment:
Please state the actual error instead of only calling it "incorrectly". For example use something like: "IPv4 DNS search entry is missing" or "ipv4.dns-search setting is missing".

Reply:
Thanks for suggestion, "IPv4 DNS search entry is missing" sounds good to me.